plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/config/GraphQLParserOptions.kt

```diff
@@ -0,0 +1,20 @@
+package com.expediagroup.graphql.plugin.gradle.config
+
+import java.io.Serializable
+
+/**
+ * Configure options for parsing GraphQL queries and schema definition language documents. Settings
+ * here override the defaults set by GraphQL Java.
+ */
+data class GraphQLParserOptions(
```

Comment:
instead of null, should those values have some defaults? i.e. when configuring the plugin I have no idea what would be the default, I have to look up the `graphql-java` code to find out the defaults

Reply:
I've set default values that match current graphql-java defaults. They are not particularly accessible in the plugin sources, and if graphql-java changes these we will no longer inherit their defaults. This is easy to reverse in the future if we need to.